docker-compose.yml

```diff
@@ -4,8 +4,6 @@ x-rust-service:
   build:
     context: .
     dockerfile: docker/rust/Dockerfile
-  depends_on:
```

Comment:
But for the development version we still need ganache, right?

Reply:
yeah, if we want to run it locally, we would need to type docker-compose up ganache-cli, stablex.

I will do more research how we could do it. If you have another suggestion, let me know

Reply:
I am not sure I understand this comment. ganache is still directly below this line.